deploy/kustomize/driver/pmem-csi.yaml

```diff
@@ -250,6 +266,8 @@ spec:
             secretName: pmem-csi-node-secrets
         - name: pmem-state-dir
           hostPath:
+            # This state directory must always be the same, regardless of the CSI driver name,
+            # because we want to pick up old volumes even when the driver name gets changed.
```

Comment:
@pohly Is this same even for the operator deployments? IIRC operator uses /var/lib/<driver-name> a state directory.

Reply:
I replaced the comment with one stating the opposite:

> This state directory must include the CSI driver name because each
>installation with a certain name must have its own, separate state.